src/procstat.c

```diff
@@ -380,6 +380,8 @@ static void fuse_readdir(fuse_req_t req, fuse_ino_t ino, size_t size, off_t off,
 
 		if (!item_registered(iter))
 			continue;
+		if (iter->flags & STATS_ENTRY_FLAG_AGGREGATOR)
```

Comment:
@rmenke-lb Your suggestion works. @gil-lb do you think this is OK with the exporter?

Reply:
@anton-lb looks like all is good from exporter side, will let you know if I encounter any issue.